generators/server/templates/src/main/java/package/web/rest/_UserJWTController.java

```diff
@@ -60,23 +60,18 @@ public UserJWTController(TokenProvider tokenProvider, AuthenticationManager auth
 
     @PostMapping("/authenticate")
     @Timed
-    public ResponseEntity authorize(@Valid @RequestBody LoginVM loginVM, HttpServletResponse response) {
+    public ResponseEntity<JWTToken> authorize(@Valid @RequestBody LoginVM loginVM, HttpServletResponse response) {
```

Comment:
ResponseEntity wrapper can be removed

Reply:
Or maybe the response Authorization header could be set in the ResponseEntity and we can remove the HttpServletResponse.

Reply:
I'd keep it, that was the point of having the proper return type in swagger docs.
I'd go for removing the HttpServletResponse and returning something like this.
```
HttpHeaders httpHeaders = new HttpHeaders();
httpHeaders.add(JWTConfigurer.AUTHORIZATION_HEADER, "Bearer " + jwt);
return new ResponseEntity<>(new JWTToken(jwt), httpHeaders, HttpStatus.OK);
```

Reply:
Your swagger doc would be OK with a direct return of JWTToken. Anyway, :+1:  for ResponseEntity + auth header